aspnetcore/fundamentals/logging/index.md

```diff
@@ -2,584 +2,351 @@
-dbug: Microsoft.AspNetCore.Mvc.ModelBinding.ParameterBinder[23]
-      Done attempting to bind parameter 'id' of type 'System.String'.
-dbug: Microsoft.AspNetCore.Mvc.ModelBinding.ParameterBinder[26]
-      Attempting to validate the bound parameter 'id' of type 'System.String' ...
-dbug: Microsoft.AspNetCore.Mvc.ModelBinding.ParameterBinder[27]
-      Done attempting to validate the bound parameter 'id' of type 'System.String'.
-info: TodoApiSample.Controllers.TodoController[1002]
-      Getting item 0
-warn: TodoApiSample.Controllers.TodoController[4000]
-      GetById(0) NOT FOUND
-info: Microsoft.AspNetCore.Mvc.StatusCodeResult[1]
-      Executing HttpStatusCodeResult, setting HTTP status code 404
-info: Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker[2]
-      Executed action TodoApiSample.Controllers.TodoController.GetById (TodoApiSample) in 32.690400000000004ms
+info: Microsoft.AspNetCore.Hosting.Diagnostics[1]
+      Request starting HTTP/2 GET https://localhost:5001/Privacy
+info: Microsoft.AspNetCore.Routing.EndpointMiddleware[0]
+      Executing endpoint '/Privacy'
+info: Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker[3]
+      Route matched with {page = "/Privacy"}. Executing page /Privacy
+info: Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker[101]
+      Executing handler method DefaultRP.Pages.PrivacyModel.OnGet - ModelState is Valid
+info: Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker[102]
+      Executed handler method OnGet, returned result .
+info: Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker[103]
+      Executing an implicit handler method - ModelState is Valid
+info: Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker[104]
+      Executed an implicit handler method, returned result Microsoft.AspNetCore.Mvc.RazorPages.PageResult.
+info: Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker[4]
+      Executed page /Privacy in 74.5188ms
 info: Microsoft.AspNetCore.Routing.EndpointMiddleware[1]
-      Executed endpoint 'TodoApiSample.Controllers.TodoController.GetById (TodoApiSample)'
+      Executed endpoint '/Privacy'
 info: Microsoft.AspNetCore.Hosting.Diagnostics[2]
-      Request finished in 176.9103ms 404
+      Request finished in 149.3023ms 200 text/html; charset=utf-8
 ```
 
+The following JSON sets `Logging:Console:LogLevel:Microsoft:Information`:
+
+[!code-json[](index/samples/3.x/TodoApiDTO/appsettings.MSFT.json)]
+
+<a name="leid"></a>
+
 ## Log event ID
 
-Each log can specify an *event ID*. The sample app does this by using a locally defined `LoggingEvents` class:
+Each log can specify an *event ID*. The sample app uses the `MyLogEvents` class tp define event IDs:
 
-[!code-csharp[](index/samples/3.x/TodoApiSample/Controllers/TodoController.cs?name=snippet_CallLogMethods&highlight=3,7)]
+<!-- Review: to bad there is no way to use an enum for event ID's -->
```

Comment:
It might be a bit much to include it in the topic, but it's actually a struct, [`EventId`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.eventid?view=dotnet-plat-ext-3.1), that gets used with these calls. There's an implicit coversion from `int`, which is why this works as shown. Here's an [example](https://github.com/dotnet/aspnetcore/blob/master/src/Hosting/Hosting/src/Internal/LoggerEventIds.cs) I found for something similar to this `MyLogEvents` class, but using `EventId` with a friendly name.